x/finality/keeper/msg_server.go

```diff
@@ -389,11 +389,6 @@ func (ms msgServer) validateActivationHeight(ctx sdk.Context, height uint64) (ui
 func (ms msgServer) EquivocationEvidence(goCtx context.Context, req *types.MsgEquivocationEvidence) (*types.MsgEquivocationEvidenceResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	// validate the evidence
-	if err := validateEvidence(req); err != nil {
```

Comment:
Moved this logic to `ValidateBasic`

Reply:
The sdk calls `ValidateBasic` prior to running the msg in runTx ? or just when it enters the mempool?